pkg/cmd/compose/compose.go

```diff
@@ -58,6 +58,21 @@ func New(client *containerd.Client, globalOptions types.GlobalCommandOptions, op
 		return false, nil
 	}
 
+	options.NetworkInUse = func(ctx context.Context, netName string) (bool, error) {
+		usedMap, err := netutil.UsedNetworks(ctx, client)
+		if err != nil {
+			return false, err
+		}
+		for _, v := range usedMap {
+			for _, net := range v {
```

Comment:
`v` here is a container slice not a network slice, right ?

Reply:
Yes，I think we should use the key as the network name

Reply:
> `v` here is a container slice not a network slice, right ?

nope, `useMap` is a map which key is namespace and value is network names.